assembly/pom.xml

```diff
@@ -227,8 +83,33 @@
                                 db-tools-ij,
                                 db-shared,
                                 db-drda,
-                                javax.servlet-api,
-                                javax.ws.rs-api
+                                db-engine,
+                                pipeline_api,
+                                splice_access_api,
+                                splice_encoding,
+                                splice_machine,
+                                splice_protocol,
+                                splice_si_api,
+                                splice_timestamp_api,
+                                utilities,
+                                memory,
+                                sketches-core,
+                                hppc,
+                                opencsv,
+                                concurrentlinkedhashmap-lru,
+                                kryo-serializers,
+                                lucene-core,
+                                hbase_pipeline-${envClassifier},
+                                hbase_storage-${envClassifier},
+                                hbase_sql-${envClassifier},
+                                splice_ee-${envClassifier},
+                                splice_backup-${envClassifier},
```

Comment:
It is not a dependency but an optional inclusion in the dependency logic (i.e. it will not blow up).  You should be able to build the parcel without it however I do not believe there is explicit testing of this experience in QA.  We now put ee libs in the public repo I believe based on requests from Visa but @wtang16 would have better knowledge on that one.